src/pybind/mgr/dashboard/frontend/cypress/support/commands.ts

```diff
@@ -28,6 +28,7 @@ Cypress.Commands.add('login', () => {
     cy.request({
       method: 'POST',
       url: 'api/auth',
+      headers: { Accept: 'application/vnd.ceph.api.v1.0+json' },
```

Comment:
Would it make sense a similar `ApiVersion` class for the front-end? I see it only happens twice, so perhaps not that much needed...

Reply:
Yes, I think it's not quite needed. And to cover the multiple versions,  all we need to do is to add all MIME types in Accept header, right?